src/index.ts

```diff
@@ -5,10 +5,13 @@ export * from "./soroban_rpc";
 export { ContractSpec } from "./contract_spec";
 
 // stellar-sdk classes to expose
-export { Server } from "./server";
+export { Server, Durability } from "./server";
 export { AxiosClient, version } from "./axios";
 export * from "./transaction";
 
+// only needed for testing
```

Comment:
once it's available in the wild, nothing prevents clients from using in their production side code, should comment just be removed or not exported?

Reply:
I can't avoid exporting because it would mean it isn't available for browser testing, unfortunately :( you're right that nothing is stopping people from using it if they want. I added a comment to that effect in 47e445e.